### PR TITLE
Updated Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
🙂

### CODE REVIEW

File: po/nl.po [hunk 1]
```diff
@@ -52,22 +52,21 @@ msgstr "Rafael Mardojai CM"
 #: data/com.rafaelmardojai.Blanket.gschema.xml:42
 msgctxt "visible-name"
 msgid "'Default'"
-msgstr ""
+msgstr "‘Standaard’"
```
Comment (rafaelmardojai) [inline po/nl.po:55]:
```suggestion
msgstr "'Standaard'"
```
Please use the same unicode character from source.

Comment (Vistaus) [reply po/nl.po]: Alright, done. 🙂

Comment (rafaelmardojai) [reply po/nl.po]: Yeah, but that single quotes are there for technical reasons, will not be visible to the user.

**Comment by rafaelmardojai** [comment]
Thanks!

File: po/nl.po (hunk 1 above)
Comment (Vistaus) [reply po/nl.po]: Ah, I see. 🙂

**Pull request merged**